### ISSUE TITLE
generate-dashboards-configmap.sh generates invalid configmap

### ISSUE THREAD

**Issue by Ludek2** [coreos/prometheus-operator#947]
**What did you do?**

Generated new grafana dashboard configmap

**What did you expect to see?**

Valid configmap

**What did you see instead? Under which circumstances?**

Invalid configmap that contains python output 

Manifest created by contrib/kube-prometheus/hack/scripts/generate-dashboards-configmap.sh:

```
apiVersion: v1
kind: ConfigMap
metadata:
  name: grafana-dashboards-0
data:
Using base prefix '/usr'
New python executable in /home/ludek.navratil/prometheus-operator/contrib/kube-prometheus/.env/bin/python3
Not overwriting existing python script /home/ludek.navratil/prometheus-operator/contrib/kube-prometheus/.env/bin/python (you must use /home/ludek.navratil/prometheus-operator/contrib/kube-prometheus/.env/bin/python3)
Installing setuptools, pip, wheel...done.
Running virtualenv with interpreter /usr/bin/python3
Collecting git+https://github.com/aknuds1/grafanalib.git@v0.4.0 (from -r requirements.txt (line 1))
  Cloning https://github.com/aknuds1/grafanalib.git (to v0.4.0) to /tmp/pip-r53vyc7p-build
Requirement already up-to-date: attrs in ./.env/lib/python3.5/site-packages (from grafanalib==0.4.0->-r requirements.txt (line 1))
Installing collected packages: grafanalib
  Found existing installation: grafanalib 0.4.0
    Uninstalling grafanalib-0.4.0:
      Successfully uninstalled grafanalib-0.4.0
  Running setup.py install for grafanalib: started
    Running setup.py install for grafanalib: finished with status 'done'
Successfully installed grafanalib-0.4.0
  aws-alb-dashboard.json: |+
    {
      "dashboard":
    {
      "__inputs": [
```

**Comment by Ludek2** [comment]
#948

**Comment by brancz** [comment]
closed by #948
